src/index.js

```diff
@@ -0,0 +1,25 @@
+import './style.css';
+
+const tasks = [
+  { description: 'wash the dishes', completed: false, index: 1 },
+  { description: 'complete To Do list project', completed: true, index: 2 },
+];
+
+function renderTasks() {
+  const todoList = document.getElementById('todo-list');
+
+  tasks.sort((a, b) => a.index - b.index);
+
+  tasks.forEach((task) => {
+    const taskItem = document.createElement('li');
+    taskItem.innerHTML += `<div class="square"></div> <p>${task.description}</p> <i class="fa">&#xf142;</i>`;
```

Comment:
- [ ] Nice one dynamically rendering the tasks dynamically, but kindly note that the `checkbox` or in your case the  `<div class="square"></div>` should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.

Reply:
Thank you for your example!